calibr8/core.py

```diff
@@ -388,6 +404,15 @@ def __init__(self, independent_key:str, dependent_key:str, *, theta_names:typing
             Most calibrations are univariate (ndim=1).
             Multivariate calibrations (ndim > 1) may need to override the `infer_independent` implementation.
         """
+        noise_mixins = [m for m in type(self).__mro__ if issubclass(m, DistributionMixin)]
+        if noise_mixins[-1] is DistributionMixin:
+            warnings.warn(
```

Comment:
Den Teil verstehe ich nicht. Wenn man z.B. das LaPlace Noise nutzt, dann ist doch trotzdem die verwendete Klasse eine Subklasse von DistributionMixin.

Reply:
`issubclass(DistributionMixin, DistributionMixin) == True`, deshalb die Strategie mit der list-comprehension und dann schaue ich nur auf `noise_mixins[-1]`, wo dann nicht mehr `DistributionMixin`, sondern eine davon erbender Typ stehen soll.

Reply:
Aber ist die Reihenfolge in `__mro__` nicht so, dass die Parent Class hinten steht. Also wäre das __mro__ von LaPlace dann wirklich bei -1 nicht auch `DistributionMixin`?

Reply:
Genau, die parent class steht hinten. Also wenn die `is DistributionMixin` ist kommt die Warnung.
![grafik](https://user-images.githubusercontent.com/5894642/142217350-bc729133-a48d-48c7-a1b7-e0cf80fc284b.png)

Allerdings habe ich vergessen `calibr8.DistributionMixin` verfügbar zu machen, das korrigiere ich noch eben

Reply:
Ja genau, deswegen bin ich verwirrt. Du willst ja nicht, dass jede Subklasse falsch ist :D